app/components/FeatureCard.tsx

```diff
@@ -20,13 +21,12 @@ export default function FeatureCard({
 	return (
 		<Card
 			style={{ background: promoted ? "#D1E8D5" : "#F2F4EF" }}
```

Comment:
#D1E8D5 is defined in theme, and #F2F4EF is not just a one-off, so I would consider adding it to the theme as well